src/libraries/Common/src/Interop/Windows/Kernel32/Interop.DeviceIoControl.cs

```diff
@@ -15,6 +15,45 @@ internal static partial class Kernel32
         // https://docs.microsoft.com/windows-hardware/drivers/ddi/ntddstor/ni-ntddstor-ioctl_storage_read_capacity
         internal const int IOCTL_STORAGE_READ_CAPACITY = 0x002D5140;
 
+        // https://learn.microsoft.com/windows/win32/api/winioctl/ni-winioctl-fsctl_set_sparse
+        internal const int FSCTL_SET_SPARSE = 0x000900c4;
+        internal readonly struct FILE_SET_SPARSE_BUFFER
+        {
+            internal FILE_SET_SPARSE_BUFFER(byte setSparse) => SetSparse = setSparse;
+            internal readonly byte SetSparse;
+        }
+
+        // https://learn.microsoft.com/windows/win32/api/winioctl/ni-winioctl-fsctl_get_integrity_information
+        internal const int FSCTL_GET_INTEGRITY_INFORMATION = 0x0009027C;
+        internal readonly struct FSCTL_GET_INTEGRITY_INFORMATION_BUFFER
+        {
+            internal readonly ushort ChecksumAlgorithm;
```

Comment:
We try to match the C/C++ definitions as close as possible. The C/C++ definition is  `WORD ChecksumAlgorithm`. It is not `const WORD ChecksumAlgorithm` that matches the definition, and the C/C++ definition does not have a constructor either.

Making these structs `readonly` is introducing style inconsistency. There are hundreds of structs definitions under src/libraries/common/src/interop that can be readonly, but they are not.

(I know you have done this change based on the feedback.)